.github/workflows/manual-build.yml

```
@@ -0,0 +1,255 @@
+          ---
+          
+          **Repository:** https://github.com/${{ github.repository }}
+          EOF
+          )
+          
+          echo "NOTES<<EOF" >> $GITHUB_OUTPUT
+          echo "$NOTES" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)